### DISCUSSION TITLE
Filter Multiple AI Transcript Windows Seperately

### DISCUSSION THREAD

**Discussion by friedcpu** [f1multiviewer/issue-tracker#456] [Feature requests] [answered]
Hi,
You can currently open multiple AI Transcript windows but when you apply a filter it filters every window the same. Would be nice if you could have different filters on the different windows. So you could select say the Red Bull drivers in 1 and the Mercedes drivers in another or whatever other way you would like to separately filter :)

**Comment by LapsTimeOFF** [comment, marked as answer]
Hey, this has been implemented in version `v2.8.0`
